assets/js/study.js

```diff
@@ -22,6 +24,12 @@ const comments_div = "comments";
 
 let combo_count = 0;
 
+let seen_suggestions = [];
```

Comment:
Can you add some documentation for `seen_suggestions` and `scheduled_suggestions`?

Reply:
I have refactored the code a bit now, so I removed one of the functions completely and the other has been documented.